Huawei/netflow-9.conf

```diff
@@ -1,12 +1,42 @@
+
+slot 1
+ip netstream sampler to slot self
+#
+slot 2
+ipv6 netstream sampler to slot self
+#
 ip netstream timeout active 1
 ip netstream timeout inactive 15
-ip netstream export version 9
-
+ip netstream export version 9 origin-as
+#
 # this command is only necessary if you run VRP >= 5.0
 # it will store interface indices in 32bit counters (as SNMP does) instead of 16bit.
 ip netstream export index-switch 32
-
 ip netstream export template timeout-rate 1
+ip netstream sampler fix-packets {{device_sample_rate}}  inbound
+ip netstream export source {IP-ADDRESS}
+ip netstream export host{{kentik_ingest_IP}} {{kentik_ingest_PORT}} vpn-instance INTERNET
+#
+ipv6 netstream timeout active 1
+ipv6 netstream timeout inactive 15
+ipv6 netstream export version 9 origin-as
+#
+ip netstream monitor FLOW-IPv4
+ip netstream export host {{kentik_ingest_IP}} {{kentik_ingest_PORT}} vpn-instance INTERNET version 9
+#
+ipv6 netstream monitor FLOW-IPv6
+#
+#
+set flow-stat interval 30
+ipv6 enable
+ip netstream inbound
+#
 ip netstream sampler fix-packets {{device_sample_rate}} inbound
-ip netstream export source {{device_sending_ip}}
-ip netstream export host {{kentik_ingest_ip_from_UI}} {{kentik_ingest_UDP_port_from_UI}}
+#  or for randomsampling:
+#  ip netstream sampler random-packets  {{device_sample_rate}} inbound
+#
+ipv6 netstream inbound
```

Comment:
this command should not be in the global configuration mode. It is supposed to be implemented at the interface?
ref: https://support.huawei.com/enterprise/en/doc/EDOC1000174046/3a962da5/netstream-configuration-commands#ip_netstream_2
you can add it to the interface file: https://github.com/kentik/config-snippets/blob/cmcgrath138-patch-1/Huawei/netflow-9-interfaces.conf